crates/uv/Cargo.toml

```diff
@@ -72,7 +72,6 @@ owo-colors = { workspace = true }
 rayon = { workspace = true }
 regex = { workspace = true }
 rustc-hash = { workspace = true }
-same-file = { workspace = true }
```

Comment:
This is genuinely unused in the crate now, so better to remove than ignore `cargo-shear`.

Reply:
Completely missed that this was factored into a separate crate, thanks for catching!